test/unit/relative-time/format.js

```diff
@@ -0,0 +1,107 @@
+define([
+	"cldr",
+	"src/relative-time/format",
+	"src/relative-time/properties",
+	"json!cldr-data/main/en/dateFields.json",
+	"json!cldr-data/supplemental/likelySubtags.json",
+
+	"cldr/event",
+	"cldr/supplemental"
+], function( Cldr, format, properties, enDateFields, likelySubtags ) {
+
+var cldr;
+
+Cldr.load( enDateFields );
+Cldr.load( likelySubtags );
+
+cldr = new Cldr( "en" );
+
+QUnit.module( "Relative Time Format" );
+
+function makeMockNumberFormatter(assert, expectedValue) {
+	return function ( value ) {
+		assert.equal( value, expectedValue );
+		return expectedValue.toString();
+	};
+}
+function makeMockPluralGenerator(plural) {
+	return function () {
+		return plural;
+	};
+}
```

Comment:
I'm wondering if we could drop `make` from these function names. I would still understand them and we'd have shorter names.